src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java

```diff
@@ -71,10 +71,18 @@
     /**
      * Build reports from parallel tests.
      *
-     * @parameter property="true" default-value="false"
+     * @parameter property="parallelTesting" default-value="false"
      * @required
      */
     private Boolean parallelTesting;
+    
+    /**
+     * Build Pass threadhold
+     *
+     * @parameter default-value="100"
+     * @required
+     */
+    private int passPercentThreshold=100;
```

Comment:
this currently is not supported by the cucumber-reporting project and if that happens this flag might be the candidate for being removed or refactored

Reply:
What is the ask here ? I have implemented the threshold limit for which this parameter is required.

Reply:
The point is that such features I'm trying to implement first in https://github.com/damianszczepanik/cucumber-reporting and then add control for dependencies projects such as this one. So would you add this kind of feature to cucumber-reporting project first? I've already found similar request from one of the user and it would be nice to have control for percentage of features, scenarios and steps

Reply:
Sure. Will implement that on cucumber-reporting  and then put the dependent code here. Thanks

Reply:
@aamol If I may recommend I would start from the steps (pending, undefined) just to satisfy needs reported in https://github.com/damianszczepanik/cucumber-reporting/pull/636 and similar. Then propagate this change to Jenkins, Maven wrappers.
Then we will get feedback soon so later you can add support for scenarios and features

Reply:
@aamol  ping